Source/WTF/wtf/text/TextBreakIterator.h

```diff
@@ -345,7 +343,7 @@ class CachedLineBreakIteratorFactory {
     StringView m_stringView;
     AtomString m_locale;
     std::optional<CachedTextBreakIterator> m_iterator;
-    const UChar* m_cachedPriorContext { nullptr };
+    const UChar* m_cachedPriorContext;
```

Comment:
What's the rationale for removing this initialization?

Reply:
My bad, I want to span and then back to pointer and forgot to add the initialization back.